fastlane_core/lib/fastlane_core/update_checker/update_checker.rb

```diff
@@ -123,9 +123,11 @@ def self.generate_fetch_url(gem_name)
     def self.ios_app_identifier(args)
       # args example: ["-a", "com.krausefx.app", "--team_id", "5AA97AAHK2"]
       args.each_with_index do |current, index|
+        # rubocop: disable Style/MultipleComparison
```

Comment:
This isn't my code, and I don't have a strong opinion. But it might be better to disable it given that `rubocop -a` doesn't seem to handle this case. I did all the changes manually in an earlier commit to satisfy this Cop. I've pushed another commit that removes these comments and disables Style/MultipleComparison in .rubocop.yml.